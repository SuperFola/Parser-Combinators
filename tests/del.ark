(del a)

(
    del b
)

(
    del
    c
)

(
del
d
)