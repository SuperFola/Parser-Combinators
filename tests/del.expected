( ( Keyword:del Symbol:a ) ( Keyword:del Symbol:b ) ( Keyword:del Symbol:c ) ( Keyword:del Symbol:d ) )